src/gol.js

```diff
@@ -0,0 +1,37 @@
+export default function next(input) {
+  const output = input.map((row) => row.map((cell) => cell));
+  /*
+    input.map(function(row) {
+      return row.map(function(cell) {
+        return cell
+      })
+    })
+  */
+
+  for (let i = 0; i < input.length; i += 1) {
+    for (let j = 0; j < input[i].length; j += 1) {
+      const liveNeighbours = countLiveNeighbours(input, i, j);
+      if (liveNeighbours == 3) {
```

Comment:
I would like to see comments around these conditions as this is basically the main business logic.